src/main/java/space/obminyashka/items_exchange/service/impl/AuthServiceImpl.java

```diff
@@ -48,9 +48,9 @@ public Optional<UserLoginResponseDto> createUserLoginResponseDto(String username
 
     @Override
     public boolean logout(String accessToken, String username) {
-        final String token = JwtTokenProvider.resolveToken(accessToken);
+        final String token = JwtTokenService.resolveToken(accessToken);
         if (!token.isEmpty()) {
-            jwtTokenProvider.invalidateAccessToken(token);
+            jwtTokenService.invalidateAccessToken(token);
```

Comment:
There no checking to null, so it could NPE happen

Reply:
It's never can be null here. In SecurityConfig we have such verification:
`.antMatchers(ApiKey.AUTH_OAUTH2_SUCCESS, ApiKey.AUTH_LOGOUT, ApiKey.USER_MY_INFO, ApiKey.USER_MY_ADV).authenticated()`
that means Spring Security won't invoke the endpoint if it wouldn't find a token within the request.